src/renderer/prefComponents/editor/index.vue

```diff
@@ -34,20 +34,20 @@
       :onChange="value => onSelectChange('codeFontSize', value)"
     ></range>
     <font-text-box
-      description="The font used in code blocks."
+      description="Font family in code blocks."
```

Comment:
```suggestion
      description="Font used in code blocks.”
```

Reply:
It depends on how precise one wants to be. The word "used" is unnecessary. "Font family" is a more precise description, since formally bold type is a different font compared to regular weight.

Reply:
It just feels a bit... off. I guess family is more *accurate*, I was just going on how it... feels(?)

Reply:
I have no overwhelming impulse to include the word "family". However, simply "Font in code blocks" strikes me as awkward, leaving me to wonder why not include the extra word that is more accurate and descriptive,  that being "family", rather than the word "used", which lacks any genuine descriptive value.

Reply:
Maybe "Font *for* code blocks"?